app/scripts/lib/AccountIdentitiesPetnamesBridge.test.ts

```diff
@@ -181,24 +181,30 @@ describe('AccountIdentitiesPetnamesBridge', () => {
     );
   });
 
-  it('does not delete entry when account identity is deleted', () => {
-    const preferencesController = createPreferencesControllerMock(
-      createPreferencesStateWithIdentity(ADDRESS_MOCK, NAME_MOCK),
-    );
-    const nameController = createNameControllerMock(
-      createNameStateWithPetname(ADDRESS_MOCK, NAME_MOCK),
-    );
-    const bridge = new AccountIdentitiesPetnamesBridge({
-      preferencesController,
-      nameController,
-      messenger: {} as any,
+  describe('shouldSyncPetname', () => {
+    it('returns true iff entry is from ACCOUNT_IDENTITY origin', () => {
+      class TestBridge extends AccountIdentitiesPetnamesBridge {
+        public shouldSyncPetname(entry: PetnameEntry): boolean {
+          return super.shouldSyncPetname(entry);
+        }
+      }
+      const preferencesController = createPreferencesControllerMock(
+        EMPTY_PREFERENCES_STATE,
+      );
+      const nameController = createNameControllerMock(EMPTY_NAME_STATE);
+      const bridge = new TestBridge({
+        preferencesController,
+        nameController,
+        messenger: {} as any,
+      });
+      bridge.init();
+      Object.values(NameOrigin).forEach((origin) => {
+        expect(
+          bridge.shouldSyncPetname({
+            origin,
+          } as any),
+        ).toBe(origin === NameOrigin.ACCOUNT_IDENTITY);
```

Comment:
Very minor, but I'm a fan of explicit logic in tests rather than conditional logic to maintain readability and to avoid duplicating the same logic from the implementation.

In this example we're doing the same `origin ===` condition so arguably these test would be more stable if we had two tests such as `returns true if origin is account identity` and `returns false if origin is not account identity`.

If we really want to test every value in the enum, would it be cleaner to use a `it.each` with a table of every origin value and if we expect it to be `true` or `false`, that ay we avoid duplicating the test logic.

Reply:
Done. Thank you for the suggestion.